Vic2ToHoI4/Source/HOI4World/HoI4Country.cpp

```diff
@@ -201,6 +201,27 @@ HoI4::Country::Country(const std::shared_ptr<Country> owner,
 }
 
 
+HoI4::Country::Country(const std::string& region_,
+	 const Regions& regions,
+	 Mappers::GraphicsMapper& graphicsMapper,
+	 Names& names):
+	 primaryCulture("unrecognized"),
+	 primaryCultureGroup("unrecognized"), unrecognizedNation(true), region(region_), oldCapital(-1)
+{
+	if (const auto& regionName = regions.getRegionName(region); regionName)
+	{
+		name_ = "Unrecognized " + *regionName;
+	}
+
+	if (const auto& regionAdjective = regions.getRegionAdjective(region); regionAdjective)
+	{
+		adjective_ = *regionAdjective;
+	}
+
+	color = commonItems::Color(std::array{128, 128, 128});
```

Comment:
all same color? Maybe a bit of jitter would help...

Reply:
I've opened #1179 to address this.